src/cloud_provider/aws/kubernetes/helm_charts.rs

```diff
@@ -1254,85 +1239,90 @@ impl HelmChart for AwsVpcCniChart {
 
         match self.enable_cni_managed_by_helm(kubernetes_config, envs) {
             true => {
-                info!("Enabling AWS CNI support by Helm");
-
                 for kind in kinds {
-                    info!("setting annotations and labels on {}/aws-node", &kind);
-                    let steps = || -> Result<(), SimpleError> {
+                    // Setting annotations and labels on kind/aws-node
+                    let steps = || -> Result<(), CommandError> {
+                        let tag = format!("meta.helm.sh/release-name={}", self.chart_info.name);
+                        let args = vec![
+                            "-n",
+                            "kube-system",
+                            "annotate",
+                            "--overwrite",
+                            kind,
+                            "aws-node",
+                            tag.as_str(),
+                        ];
+                        let mut stdout = "".to_string();
+                        let mut stderr = "".to_string();
+
                         kubectl_exec_with_output(
-                            vec![
-                                "-n",
-                                "kube-system",
-                                "annotate",
-                                "--overwrite",
-                                kind,
-                                "aws-node",
-                                format!("meta.helm.sh/release-name={}", self.chart_info.name).as_str(),
-                            ],
+                            args.clone(),
                             environment_variables.clone(),
-                            |_| {},
-                            |_| {},
+                            |out| stdout = out.to_string(),
```

Comment:
Need to concat strings, or it will just keep the last stdout/error line

Reply:
Done